docs/version0.41.md

```diff
@@ -44,9 +45,15 @@ A VM maintains multiple platform threads that are used as carrier threads to run
 
 For more information, see [`-XX:[+|-]ShowCarrierFrames`](xxshowcarrierframes.md). ![End of content that applies to Java 21 (LTS) and later](cr/java_close_lts.png)
 
+### ![Start of content that applies to Java 21 (LTS) and later](cr/java21plus.png) New `-XX:ContinuationCache` option added
+
+When a virtual thread is created, it is associated with a Continuation object. A virtual thread saves its current state in its continuation object and later resumes from that state. When the continuation implementation is complete, instead of freeing the allocated memory, the VM stores such a memory in a continuation cache. To create a new continuation object, the VM can directly use the cached structure.
```

Comment:
please modify the wording here, Continuation object is different from the cache.
In OpenJ9's implementation, there are multiple levels of virtual thread data structure:
```
VirtualThread Object (User level Java Object)
-> Continuation Object (VM level Java Object)
    -> J9VMContinuation structure (VM level native structure)
        -> internal fields/structure contained within J9VMContinuation structure
```
The caching is done at J9VMContinuation level, which doesn't affect Java object allocation.

```suggestion
When a virtual thread is created, it is associated with a Continuation object which holds the target task of the virtual thread. When the virtual thread is started, the VirtualMachine stores the current runtime state in internal allocated structure linked with the Continuation object and can later resumes execution from that state. When the Continuation task is complete, instead of freeing the allocated memory, the VM stores such structure as a continuation cache. New Continuation objects can directly use the cached structure instead of reallocating in memory.
```

same apply to the option explanation page